FASTER/ViewModel/SteamUpdaterViewModel.cs

```diff
@@ -160,6 +160,7 @@ public async Task UpdateClick()
                 {233793, "Arma 3 Server Creator DLC - CSLA"},
                 {233794, "Arma 3 Server Creator DLC - SOGPF"},
                 {233795, "Arma 3 Server Creator DLC - WS"},
+                {2647760, "Arma 3 Server Creator DLC - RF"},
```

Comment:
Is build number, Depot Id hasn't been released. Will be updated on Release of RF